README.md

```diff
@@ -19,6 +19,10 @@ Language : English | [中文](./README.zh-CN.md)
 
 For more information please refer to [Installation Guide](./INSTALLATION.md).
 
+## Datasource
+
+`Hypercrx` consumes data that generated by [OpenDigger](https://github.com/X-lab2017/open-digger), which is an amazing open source project that focuses on open source analysis.
```

Comment:
I think you can remove `amazing`, this is not an objective word, lol.

Reply:
OK 🤣